From 4da1695a522264515ccbb59a142254efcfd5b864 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 04 Jul 2018 16:02:17 +0300
Subject: [PATCH] board: habana: switch default to outlet #4

Now that we have the same power source with monster5, switch to outlet #4.
Note (once again), this probably should somehow be defined in runtime/external
configuration.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/ip_pdu b/board/habanalabs/common/ip_pdu
index a162367..1cf0479 100755
--- a/board/habanalabs/common/ip_pdu
+++ b/board/habanalabs/common/ip_pdu
@@ -13,7 +13,7 @@
 
 case "$board" in
 *asic*)
-	pwr_id=5
+	pwr_id=4
 	;;
 *onix)
 	pwr_id=1
